src/SpeedJumpIrm.sol

```diff
@@ -120,43 +124,39 @@ contract AdaptativeCurveIrm is IIrm {
 
         int256 startRateAtTarget = rateAtTarget[id];
 
-        // First interaction.
         if (startRateAtTarget == 0) {
+            // First interaction.
+            // Safe "unchecked cast" because INITIAL_RATE_AT_TARGET >= 0.
```

Comment:
[minor] let's use `Safe "unchecked" cast` everywhere (there are 3 places to change)